Could not find `btree_map` in `alloc`

Hmm ok filing an issue since I don't quite understand the interaction here...
If I compile goblin 8bbbcf5 directly with `cargo +nightly build --no-default-features --features alloc` it works fine.

If I try to compile the object crate with the goblin dependency declared as follows:
```
[dependencies.goblin]
git = "https://github.com/m4b/goblin"
default-features = false
features = ["alloc", "endian_fd", "elf32", "elf64", "mach32", "mach64", "pe32", "pe64", "archive"]
```
Using `cargo +nightly build --no-default-features` or `cargo +nightly build -v --no-default-features --features goblin/alloc`, I get:

```
   Compiling goblin v0.0.16 (https://github.com/m4b/goblin#8bbbcf5d)
error[E0432]: unresolved import `alloc::btree_map`==========>          ] 10/12: goblin
  --> /Users/gz/.cargo/git/checkouts/goblin-d0c041c0a85ca4ca/8bbbcf5/src/archive/mod.rs:15:12
   |
15 | use alloc::btree_map::BTreeMap;
   |            ^^^^^^^^^ Could not find `btree_map` in `alloc`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: Could not compile `goblin`.

To learn more, run the command again with --verbose.
```

The following diff https://github.com/gz/goblin/commit/a6369aced1a09e8a2c2f75e347491e2c75fd97cc fixes the problem.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - 1.18.0
+  - 1.19.0
   - stable
   - beta
   - nightly

--- a/README.md
+++ b/README.md
@@ -10,7 +10,7 @@ https://docs.rs/goblin/
 
 ### Usage
 
-Goblin requires `rustc` 1.18.
+Goblin requires `rustc` 1.19.
 
 Add to your `Cargo.toml`
 

--- a/src/archive/mod.rs
+++ b/src/archive/mod.rs
@@ -12,7 +12,7 @@ use strtab;
 use error::{Result, Error};
 
 use core::usize;
-use alloc::btree_map::BTreeMap;
+use alloc::collections::btree_map::BTreeMap;
 use alloc::vec::Vec;
 
 pub const SIZEOF_MAGIC: usize = 8;

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -102,7 +102,7 @@ mod alloc {
     pub use std::boxed;
     pub use std::string;
     pub use std::vec;
-    pub use std::collections::btree_map;
+    pub use std::collections;
 }
 
 /////////////////////////

